engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/AvroSchemaRegistry.scala

```diff
@@ -0,0 +1,27 @@
+package pl.touk.nussknacker.engine.avro
+
+import org.apache.avro.Schema
+import org.apache.flink.api.common.typeinfo.TypeInformation
+import org.apache.flink.streaming.api.functions.TimestampAssigner
+import pl.touk.nussknacker.engine.kafka.RecordFormatter
+import pl.touk.nussknacker.engine.kafka.serialization.{DeserializationSchemaFactory, SerializationSchemaFactory}
+
+trait SchemaRegistryClient extends Serializable {
+  def schemaById(id: Int): Schema
+  def schemaBySubjectAndVersion(subject: String, version: Int): String
+  def latestSchema(name: String): String
+}
+
+trait SchemaRegistryProvider[T] {
```

Comment:
Name is misleading. Regards to it I expect that I will find only schema registry client here. Why you've put all together?

Reply:
You're right - changed.